### PR TITLE
Add function Deploy-FinOpsHub

### PR DESCRIPTION
**Pull request by aromano2**
## 🛠️ Description

Fixes #123 

## 🔬 How has this been tested?

- [ ] 🫰 PS -WhatIf / az validate
- [x] 👍 Manually deployed + verified
- [ ] 💪 Unit tests

### CODE REVIEW

File: src/powershell/FinOpsToolkit.psm1 [hunk 1]
```diff
@@ -151,39 +147,183 @@ function Get-FinOpsToolkitVersion
...
+        [string]
+        $Name,
+
+        [Parameter(Mandatory = $true)]
+        [string]
+        $ResourceGroup,
+
+        [Parameter()]
+        [string]
+        $Location = 'westus',
```
[the remark this answers is not in the document]
Comment (aromano2) [reply src/powershell/FinOpsToolkit.psm1]: fixed

File: src/powershell/FinOpsToolkit.psm1 [hunk 2]
```diff
@@ -151,39 +147,181 @@ function Get-FinOpsToolkitVersion
         $Preview
     )
 
+    $progress = $ProgressPreference
+    $ProgressPreference = 'SilentlyContinue'
     $releaseUri = 'https://api.github.com/repos/microsoft/cloud-hubs/releases'
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:152]:
```suggestion
    $releaseUri = 'https://api.github.com/repos/microsoft/finops-toolkit/releases'
```

File: src/powershell/FinOpsToolkit.psm1 [hunk 3]
```diff
@@ -151,39 +147,181 @@ function Get-FinOpsToolkitVersion
...
+        Name of the FinOps hub instance.
+
+    .PARAMETER ResourceGroup
+        Name of the resource group to deploy to. Will be created if it doesn't exist.
+
+    .PARAMETER Location
+        Azure location to execute the deployment from.
+
+    .PARAMETER Version
+        Optional. Version of FinOps hub template to use. Defaults to latest.
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:212]: Aligning to what we do in Bicep...
```suggestion
        Optional. Version of FinOps hub template to use. Defaults = "latest".
```

File: src/powershell/FinOpsToolkit.psm1 [hunk 4]
```diff
@@ -151,39 +147,181 @@ function Get-FinOpsToolkitVersion
...
 
-    $output = @()
-    foreach ($release in $releases)
+    .PARAMETER Preview
+        Optional. Deploy a pre-release version of FinOps Hub.
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:215]: I believe we usually say "Indicates..." for boolean values. Also adding in context about version.
```suggestion
        Optional. Indicates that a pre-release version of FinOps hub can be used when -Version is "latest".
```

File: src/powershell/FinOpsToolkit.psm1 [hunk 5]
```diff
@@ -151,39 +147,181 @@ function Get-FinOpsToolkitVersion
...
+
+    .PARAMETER StorageSku
+        Optional. Storage account SKU. Premium_LRS = Lowest cost, Premium_ZRS = High availability. Note Standard SKUs are not available for Data Lake gen2 storage.
```
Comment (flanakin) [inline src/powershell/FinOpsToolkit.psm1:218]:
```suggestion
        Optional. Storage account SKU. Premium_LRS = Lowest cost, Premium_ZRS = High availability. Note Standard SKUs are not available for Data Lake gen2 storage. Default = "Premium_LRS".
```

**Review by flanakin — APPROVED** [review]

**Pull request merged**